src/renderers/WebGLRenderer.js

```diff
@@ -1547,9 +1547,13 @@ function WebGLRenderer( parameters ) {
 
 				var shader = ShaderLib[ parameters.shaderID ];
 
+				var computedUniforms = undefined !== material.shaderUniforms ?
```

Comment:
if these are present, choose which uniforms to pass
**pattern from how `.defines` worked**